frontend/src/components/button/NavButtonWrapper.tsx

```diff
@@ -1,17 +1,13 @@
 import styles from '@/styles/NavButtonWrapper.module.css';
 import Image from 'next/image';
 import Link from 'next/link';
-
-// home button
-type HomeButtonProps = {
-  onClickFunction: () => void;
-};
+import { HomeButtonProps, ListButtonProps, PublicChannelListButtonProps, GameButtonProps } from '@/type';
 
 const HomeButton = ({ onClickFunction }: HomeButtonProps) => {
   return (
     <button>
       <Link href="/" style={{ position: 'relative', width: '100%', height: '100%' }}>
-        <Image src="/assets/home-button.png" alt="home-button" onClick={onClickFunction} fill style={{ objectFit: 'contain',}} />
+        <Image src="/assets/home-button.png" alt="home-button" onClick={onClickFunction} fill style={{ objectFit: 'contain', }} />
```

Comment:
```suggestion
        <Image src="/assets/home-button.png" alt="home-button" onClick={onClickFunction} fill style={{ objectFit: 'contain' }} />
```
style의 인자값으로 `,`이 의도된게 아니라면 지우는게 좋아보입니다!